src/Arcus.Scripting.KeyVault/Scripts/Set-AzKeyVaultSecretFromFile.ps1

```diff
@@ -32,4 +33,4 @@ if ($Expires -ne $null) {
 }
 
 $version = $secret.Version
-Write-Host "Secret '$SecretName' (Version: '$version') has been created."
+Write-Host "Azure Key Vault Secret '$SecretName' (Version: '$version') has been created."
```

Comment:
Can you update the docs as well, to reflect these updates in output?

Reply:
Sure, thanks!